docs/modules/ROOT/pages/how-to/customize-cypher.adoc

```diff
@@ -125,16 +127,18 @@ WHERE this0.name = $myParam
 RETURN this0
 ----
 
-Note that `$myParam` does not return as a param by `.build()`. 
+Note that `$myParam` does not return as a param by `.build()`, as its value has not been defined. 
 To generate the parameter, pass a value in the same way as normal parameters:
 
 [source, javascript]
 ----
 const movie = new Cypher.Node();
-const matchQuery = new Cypher.Match({movie,   labels: ["Movie"] }).where(movie, { name: new Cypher.NamedParam("myParam", "Keanu Reeves") }).return(movie);
+const matchQuery = new Cypher.Match({movie,labels: ["Movie"] })
+    .where(movie, { name: new Cypher.NamedParam("myParam", "Keanu Reeves") })
+    .return(movie);
 ----
 
-The resulting parameter is:
+The resulting parameters are:
```

Comment:
Changing it to plural but the example is still a singular param?

Reply:
well, it is a single param, but there could be more :thinking:

Reply:
I suggest either the example includes multiple parameters or you leave it as singular instead of plural.

Reply:
I made a small change here to make it clearer